profile_library/signify/LTD011/model.json

```
@@ -0,0 +1,14 @@
+{
+  "calculation_strategy": "lut",
+  "created_at": "2025-03-12T22:12:11.404691",
+  "measure_description": "Measured with utils/measure script",
+  "measure_device": "TP-Link Kasa HS110",
+  "measure_method": "script",
+  "measure_settings": {
+    "SAMPLE_COUNT": 2,
+    "SLEEP_TIME": 3,
+    "VERSION": "v1.17.9:docker"
+  },
+  "name": "Phillips Hue Garnea Downlight",
```

```suggestion
  "name": "Hue Garnea Downlight",
```

There is a typo in Philips, but the other hue lights we just start with Hue in the name, so I prefer this way to be consistent.